src/applications/check-in/hooks/useGetCheckInData.jsx

```diff
@@ -182,8 +184,7 @@ const useGetCheckInData = ({
             return;
           }
 
-          if (preCheckinAlreadyCompleted(payload.appointments)) {
-            setPreCheckinComplete(window, true);
```

Comment:
Are you sure that already completed shouldn't also go to complete page? I don't see the success alert on appointments page scenario in the unified wires anymore.

Reply:
Yeah I don't see that in the new wires either. I just left it in because that's the way that scenario was working. I think it makes sense just to take them back to the complete page.